api/models/workshop.py

```diff
@@ -23,6 +25,10 @@ class Workshop(CapacityMixin, Base):
         WorkshopDifficulty,
         verbose_name=_("Difficulty"),
     )
+    location = models.ForeignKey(
```

Comment:
Do we need to keep track of workshop over the years? I would say yes. I think, that we need to say, what lector had a workshop last year. What was the participants of that workshop. What was the content.

If we reuse workshop for a new year we either lose all that information from last year or we need to keep track of all the changes.

If we keep track of all that information (as suggested by linking workshop to different location over the years), we would need to store all the old descriptions, lectors, participants and other. That is a LOT of changes.

I propose, we don't reuse `Workshop` in a new year. We could copy them, but the old `workshop`s should stay as record of the past year.